roq-management/src/main/java/org/roqmessaging/management/server/state/ExchangeState.java

```diff
@@ -0,0 +1,5 @@
+package org.roqmessaging.management.server.state;
+
+public class ExchangeState {
+
+}
```

Comment:
I don't think this is used. I think the one you really use is _roq-core/src/main/java/org/roqmessaging/state/ExchangeState.java_